selfdrive/car/nissan/carstate.py

```diff
@@ -49,13 +49,16 @@ def update(self, cp, cp_adas, cp_cam):
     if self.CP.carFingerprint in [CAR.ROGUE, CAR.XTRAIL]:
       ret.seatbeltUnlatched = cp.vl["HUD"]["SEATBELT_DRIVER_LATCHED"] == 0
       ret.cruiseState.available = bool(cp_cam.vl["PRO_PILOT"]["CRUISE_ON"])
-    elif self.CP.carFingerprint == CAR.LEAF:
-      ret.seatbeltUnlatched = cp.vl["SEATBELT"]["SEATBELT_DRIVER_LATCHED"] == 0
+    elif self.CP.carFingerprint in [CAR.LEAF, CAR.LEAF_IC]:
```

Comment:
this is unnecessary, just make an elif for each leaf

Reply:
I would then have to duplicate line 57 for each leaf. If you prefer this, I will implement it like that.

Reply:
missed that